### PR TITLE
ensure project name starts with alpha char

### LINKED ISSUE #957: Projects should start with an alpha char
### Description

Following on from discussions around https://github.com/flowforge/flowforge-driver-k8s/pull/53, we should tighten up the creation of projects and only permit project names starting with an alpha char `a-zA-Z` only

### Epic/Story

_No response_

### PR DESCRIPTION
**Pull request by Steve-Mcl**
fixes #957

### CODE REVIEW

File: frontend/src/pages/team/createProject.vue [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default {
     },
     watch: {
         'input.name': function (value, oldValue) {
-            if (/^[a-zA-Z0-9-]+$/.test(value)) {
+            if (/^[a-zA-Z]+[a-zA-Z0-9-]*$/.test(value)) {
```
[the remark this answers is not in the document]
Comment (Steve-Mcl) [reply frontend/src/pages/team/createProject.vue]: Unnecessary quantifier removed in [c90d414](https://github.com/flowforge/flowforge/pull/958/commits/c90d414d927c9fe0047da7d41202fbfcb5600d9e)

**Comment by Steve-Mcl** [comment]
@hardillb ready for (re)review please

**Review by hardillb — APPROVED** [review]

**Comment by Steve-Mcl** [comment]
e2e test fail is odd...

```
 FlowForge - Team Membership
    ✓ loads team members into the data table (4026ms)
    ✓ member cannot can invite or remove a user (3647ms)
    1) admin/owner can remove a member from the team
    ✓ owner/admin can invite user (3930ms)
    ✓ loads invitations into the data table (3087ms)
    ✓ user can accept a team invite (4924ms)


  5 passing (28s)
  1 failing

  1) FlowForge - Team Membership
       admin/owner can remove a member from the team:
     AssertionError: Timed out retrying after 4000ms: Not enough elements found. Found '0', expected '2'.
      at Context.eval (http://localhost:3000/__cypress/tests?p=test/e2e/frontend/cypress/tests/team-membership.spec.js:133:58)
```

Gonna merge main -> PR as there have been changes in this area.

**Comment by hardillb** [comment]
I didn't merge it as I was just looking at the tests

**Comment by Steve-Mcl** [comment]
> I didn't merge it as I was just looking at the tests

I've merged main & ran e2e locally...

image


Lets see how GH fairs after I sync up

**Comment by Steve-Mcl** [comment]
@hardillb yeah - ignore that the test is failing after re-building...

image


Looking into it now.

**Pull request merged**